app/src/main/java/timur/gilfanov/messenger/ui/screen/user/SettingsUI.kt

```diff
@@ -0,0 +1,3 @@
+package timur.gilfanov.messenger.ui.screen.user
+
+data class SettingsUI(val language: String)
```

Comment:
For naming consistency:
```suggestion
data class SettingsUi(val language: String)
```